web/impact/impact/tests/test_analyze_judging_round_view.py

```diff
@@ -96,7 +100,7 @@ def test_get_with_unread_application(self):
                     context.needed_reads())
 
     def test_get_with_gender_criterion(self):
-        genders = ["f", "m"]
+        genders = ["female", "male"]
```

Comment:
Huh. This should fail, since we never store "female" or "male" in the db.

Reply:
this actually created female and male in the db through the AnalyzeJudgingContext which is why it still works

Reply:
Looked into this. It actually makes sense: if we specify bad criteria, it just doesn't register any reads for them which is what the test ultimately asserts. 
Restoring these to the expected values, leaving it alone otherwise.